openshift/templates/iiw-book-service/iiw-book-service-deploy.dev.param

```diff
@@ -0,0 +1,35 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../openshift/templates/iiw-book-service/iiw-book-service-deploy.yaml
+#=========================================================
+# NAME=iiw-book-service
+# APP_GROUP=iiw-book
+NAMESPACE_NAME=a99fd4
+# ROLE=service
+# AGENT_SERVICE_NAME=iiw-book-agent
+APPLICATION_DOMAIN=iiw-book-service-dev.apps.silver.devops.gov.bc.ca
```

Comment:
For consistency with the naming pattern, I would suggest making the `APPLICATION_DOMAIN` urls look like `iiwbook-dev.apps.silver.devops.gov.bc.ca`. Additionally, the URL in `APPLICATION_DOMAIN` and in `SITE_URL` need to match.

Applies to other param files as well.